nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -808,11 +808,6 @@ in
         touch /etc/NIXOS
         ${config.nix.package.out}/bin/nix-env -p /nix/var/nix/profiles/system --set /run/current-system
       '';
-
-    # Add vfat support to the initrd to enable people to copy the
-    # contents of the CD to a bootable USB stick.
-    boot.initrd.supportedFilesystems = [ "vfat" ];
```

Comment:
We should keep this. While all the images produced by nixpkgs get vfat via `profile/base.nix`, external consumers can (and do) still pull in `iso-image.nix` without `base.nix` and it would be good to continue supporting the USB use case for such images.

Reply:
Fixed.